iOS_SDK/OneSignalSDK/UnitTests/UIApplicationDelegateSwizzlingTests.m

```diff
@@ -7,6 +7,8 @@
 #import "TestHelperFunctions.h"
 #import "OneSignalAppDelegateOverrider.h"
 
+#define ONESIGNALApplicationDelegate UIApplicationDelegate
```

Comment:
I don't understand the reason for this new `ONESIGNALApplicationDelegate` type. Why not use ` UIResponder<UIApplicationDelegate>` directly like the `AppDelegateForwardReceiver` below does?

Reply:
It is not necessary, but I added it as an extra layer of complexity since this is what [Google's libraries do ](https://github.com/google/GoogleUtilities/blob/4446686bc3714d49ce043d0f68318f42ed718cb6/GoogleUtilities/AppDelegateSwizzler/Public/GoogleUtilities/GULApplication.h#L24).

Reply:
ah gotcha, I missed that